@shared/api/helpers/getIconUrlFromIssuer.ts

```diff
@@ -78,28 +78,20 @@ export const getIconUrlFromIssuer = async ({
 
   if (toml.CURRENCIES) {
     /* If we find some currencies listed, check to see if they have the currency we're looking for listed */
-    toml.CURRENCIES.every(
-      async ({
-        code: currencyCode,
-        image,
-      }: {
-        code: string;
-        image: string;
-      }) => {
-        if (currencyCode === code && image) {
-          /* We found the currency listing in the toml. 3. Get the image url from it */
-          iconUrl = image;
-          /* And also save into the cache to prevent having to do this process again */
-          await sendMessageToBackground({
-            assetCanonical: `${code}:${key}`,
-            iconUrl,
-            type: SERVICE_TYPES.CACHE_ASSET_ICON,
-          });
-          return false;
-        }
-        return true;
-      },
-    );
+    toml.CURRENCIES.every(async ({ code: currencyCode, image }) => {
```

Comment:
making this async

Reply:
This code diff doesn't look like it changes the async qualities of this code. It's already async, this just looks like different formatting. Is that correct?

Reply:
Oh I see, there is a difference. I have no idea what that difference means though 😅.

Reply:
@piyalbasu  ya I'm also confused here 🤔 . Is the only difference adding the return type in the async func? But isn't it returning the same thing?

Reply:
Apologies all - this is a few levels of bad engineering on my part. I chipped away at this PR over the course of weeks and lost track of what I was doing and made an erroneous comment 😞 And then, doubly bad, I let this PR sit here and did not address the comments. Very sorry about that - textbook bad PR hygiene.

The change is *not* making this function async, as you both noted. The change is removing the type because it is now inferring the type from toml.CURRENCIES which has `code` and `image` as optional params instead of required params like the type I removed described